### ISSUE TITLE
Missing "Back to results" link

### ISSUE THREAD

**Issue by blairlearn** [NCIOCPL/r4r-app#20]
### Issue description
After transitioning from a search page to the details of an individual resource, the details page is missing the "Back to results" link that should appear adjacent to the "Resources for Researchers Home" link.

Browsing through the application with the developer tools open to the Network tab reveals that the app is, for some clicks, performing a page load instead of executing as a single page application.

> **ESTIMATE** TBD

### Steps to reproduce the issue

1. Browse to https://www.cancer.gov/research/resources/
2. Open the developer tools to the Network tab.
3. Filter for Document pages only
4. Under "Research Area", click "Cancer Biology"
5. Click the link for "Algorithm for the Reconstruction of Accurate Celluar Networks (ARACNe)"
   - You can actually use any of the links.

### What's the expected result?
- The network tab should not show any traffic (this is why we filtered it)
- Immediately below the title, there should be a "Back to results" link to the left of the "Resources for Researchers Home" link.

### What's the actual result?
- The network tab shows a GET request for `https://www.cancer.gov/research/resources/resource/250`
- The "Back to results" link is missing

### Additional details / screenshot
- This behaviro can be reproduced on production, lower tiers, and also with a local Drupal build.
- With some experimentation, it's possible to get the correct page (with "Back to results" link) using archive.org.
- 
![image](https://github.com/user-attachments/assets/55ae8981-0fea-409d-b4cc-21bb0b065b36)
- 
![image](https://github.com/user-attachments/assets/618f2962-06b7-4e53-bb4c-587e11847810)

### Related Tickets
* NCIOCPL/clinical-trials-search-app#633

**Comment by bryanpizzillo** [comment]
In trying to make R4R generic, I noticed this issue. This is because of a Launch click handler for tracking “legacy components” on mini landing pages.

Basically we thought that if we had a condition for the page type (cgvMiniLanding) that the click event handler would not be attached to items outside of that type. Unfortunately, the click handler is added to all links matching the css selector, and handling the event it makes sure it is a mini landing page.

The unfortunate part is in order to ensure the analytics were handling clicks to other pages we had to use a launch setting for the event that delays the event handling. Adobe basically prevents default, adds a timeout, and when the timeout finishes it sets window.location. Because this happens like this, that link becomes a whole new page load, which reloads the react app, which means the state goes away. For R4R the app’s state is where the search information is stored. For CTS it should be the query params, but there is probably something in the state that is not noticing.

So for R4R I am 100% sure this is the issue. For CTS we would have to check after the Adobe rule is disabled.
